src/cosim/timer.cpp

```diff
@@ -37,6 +36,12 @@ class real_time_timer::impl
         const auto newHash = std::hash<real_time_config>()(*config_);
         if (newHash != configHashValue_) {
             start(currentTime);
+            auto samping_period = config_->sampling_period_to_monitor.load();
```

Comment:
Typo: `samping_period` should be `sampling_period`. Also, the variable should be `const`.

Reply:
Fixed.